platform/package.json

```diff
@@ -47,7 +47,7 @@
   },
   "devDependencies": {
     "babel-core": "^6.26.3",
-    "node-sass": "^4.9.4"
```

Comment:
Parcel defaults to `dart-sass` when node-sass is not specifically included. Node-sass fails to build when there are multiple scss includes in JS. Hangs without error messages even with verbose logging. Fixed and successfully deployed when `node-sass` dep is removed